docs/email-delivery.md

```diff
@@ -53,6 +53,8 @@ Email logs are [available within the Altis Dashboard](./dashboard/logs.md) conta
 
 ## Acceptable Usage
 
+Maximum attachment size is 10mb.
```

Comment:
```suggestion
The maximum attachment size is 10MB.
```

Reply:
A slight change to the wording. Plus, I would move it to below the "Altis can send..." paragraph.